backend/src/news/utils.py

```diff
@@ -1,32 +1,51 @@
 from bs4 import BeautifulSoup
 import requests
 import json
+
+from ..database import SessionLocal
+from ..crawler.udn_crawler import UDNCrawler
+
+crawler = UDNCrawler()
     
+def convert_news_to_dict(news):
+    return {
+        "url": news.url,
+        "title": news.title,
+        "time": news.time,
+        "content": news.content,
+    }
+
```

Comment:
This function works well! 🎉
There's another way to convert news to dict. `News` inherits the `BaseModel` from Pydantic, which supports serialization/deserialization. As a result, we can get dict like this:
```py
news.model_dump()
```